model/news.model.js

```diff
@@ -42,16 +42,30 @@ exports.fetchArticleComments = (article_id) => {
                 message: "not found"
             })
         }
-    }).then((result) => {
+    }).then(() => {
         return db.query(`SELECT * FROM comments
     WHERE comments.article_id = $1
     ORDER BY created_at DESC`, [article_id]).then((result) => {
-
-
             return result.rows
         })
 
     })
+}
 
+exports.postCommentModel = (article_id, commentObj) => {
+    const {
+        username,
+        body
+    } = commentObj
+    if (Object.keys(commentObj).length === 0 || Object.keys(commentObj).length > 2) {
+        return Promise.reject({
+            status: 404,
+            message: 'not found'
+        })
```

Comment:
This should be a 400 error as it is user error if the new comment object is empty. You can achieve this by refactoring your if statement to check if `commentObj` has a `username` and `body` property. If it doesn't, promise reject that shizzle with a 400!